promscale/installation/helm.md

```diff
@@ -66,19 +69,16 @@ can provide the database URI, or specify connection parameters.
     ```bash
     helm repo update
     ```
-1.  Install the Promscale Helm chart:
-    ```bash
-    helm install --name my-release .
-    ```
-1.  Open the `values.yaml` configuration file, and locate the `connection`
+1.  Open the `values.yaml` configuration file, download the tobs
+    `values.yaml` from [here][tobs-values-yaml] and locate the `connection`
```

Comment:
@Loquacity I have marked as unresolved. Nothing was changed in the recent changed.

Reply:
@JamesGuthrie when you say "tobs hasn't been updated to account for that" you mean the instructions or that tobs (our tool) needs to be updated but it hasn't?

What would be your suggestion for this that we provide an example of values.yaml file?

Reply:
As far as I know, tobs hasn't been updated for Promscale 0.7.0, it's on 0.6.2. Correspondingly the structure of the connection block in tobs' `values.yaml` has not been updated for 0.7.0 yet. My suggestion would be that we provide the simplest possible `values.yaml` contents here instead of sending them off somewhere else.